drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -515,6 +517,136 @@ static void usbd_enable_endpoints(struct nrf_usbd_ctx *ctx)
 	}
 }
 
+/**
+ * @brief Reset endpoint state.
+ *
+ * Resets the internal logic state for a given endpoint.
+ *
+ * @param[in]  ep_cts   Endpoint structure control block
+ */
+static inline void ep_ctx_reset(struct nrf_usbd_ep_ctx *ep_ctx)
```

Comment:
is there any need for inline here, maybe it's better to let compiler optimize it as it thinks best